.gitignore

```diff
@@ -266,4 +266,6 @@ __pycache__/
 
 # /Build/
 /Build/logs
-/Build/out
\ No newline at end of file
+/Build/out
+# dotnet run does not set the working directory of the application to AnnoDesigner/bin/[Debug|Release], its sets the working directory to the root of the csproj file
```

Comment:
Minor typo here - should be `it` rather than `its` (I realise this was my own typo from a previous comment - sorry!). Could also add backticks around `dotnet run` to show that its a command - e.g \`dotnet run\`